db/testing.py

```diff
@@ -77,3 +78,22 @@ def load_low_level_data(self, mbid):
         """
         with open(self.data_filename(mbid)) as json_file:
             db.data.submit_low_level_data(mbid, json.loads(json_file.read()), gid_types.GID_TYPE_MBID)
+
+    def load_fake_low_level_data(self, mbid):
```

Comment:
This is a handy method. good idea.
We should probably make the method `submit_fake_low_level_data`, instead of `load_`
Can you add a small docstring explaining what it does?

I don't think it's important to have average loudness a random variable. Why did you make it like this?